### ISSUE TITLE
Clippy warning inside macro, docs for unsafe trait missing Safety section

### ISSUE THREAD

**Issue by sporkmonger** [bytecodealliance/wit-bindgen#922]
This is happening with the 0.24.0 release, not sure about other versions. Because it's inside the macro, I couldn't see an obvious way to ignore the warning w/ the usual method. This is a warning but I have warnings upgraded to errors in our CI.

```
error: docs for unsafe trait missing `# Safety` section
  --> crates/wasm-sdk/src/lib.rs:8:5
   |
8  | /     wit_bindgen::generate!({
9  | |         world: "bulwark:plugin/platform",
10 | |     });
   | |______^
```

For the time being I ended up just disabling it on the command line, but it's likely to also affect anyone using our SDK as it also triggers for our own macros which are wrapping the `generate!` macro.

**Comment by sporkmonger** [comment]
Created a PR to fix.

**Comment by sporkmonger** [comment]
Closed by #926.
